submissions/denismaysov/multiPopup/index.html

```diff
@@ -0,0 +1,346 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="page__body">
+      <header class="header">
+        <div class="logo">
+          <a href="https://github.com/denismaysov" target="_blank">
+            <img src="img/user.svg" alt="#"
+          /></a>
+        </div>
+        <div class="search">
+          <input
+            placeholder="Search.."
+            class="search__input"
+            id="search"
+            type="search"
+          />
+          <label class="search__label" for="search">
+            <button class="search__button" type="submit" id="search">
+              Поиск
+            </button>
+          </label>
+        </div>
+      </header>
+      <div class="main">
+        <div class="main__row">
+          <nav class="option">
+            <div class="option__wrapper">
+              <ul class="option__menu">
+                <li class="option__menu-item">
+                  <label class="option__label" for="menu__popup">
+                    <img
+                      class="option__img"
+                      src="img/menu.svg"
+                      alt="#"
+                      width="32"
+                      height="32"
+                    />
+                  </label>
+                  <input
+                    class="option__checkbox"
+                    id="menu__popup"
+                    type="checkbox"
+                  />
+                  <div class="popup">
+                    <div class="popup__container">
+                      <div class="popup__menu">
+                        <div class="popup__menu-item">
+                          <h2>First</h2>
+                          <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing
+                            elit. Nihil, iusto.
+                          </p>
+                          <p>
+                            Lorem, ipsum dolor sit amet consectetur adipisicing
+                            elit. Odio officiis autem accusantium, laudantium
+                            accusamus fugit aperiam quis at id dolores.
+                          </p>
+                          <div class="popup__menu-image">
+                            <label class="popup__menu-label" for="popup-image">
+                              <div class="popup__menu-click">
+                                <a
+                                  target="_blank"
+                                  class="popup__menu-link"
+                                  aria-label="Lofi-music radio"
+                                  href="https://www.youtube.com/watch?v=jfKfPfyJRdk"
+                                >
```

Comment:
Don't use one interactive element inside another, it's invalid markup

Reply:
how i should add more interactive elements in popup? Without a?

Reply:
You put a link inside `label`) and in this case, you have one interactive element inside another, it confusing for screen readers and browsers.